### PR TITLE
PREAPPS-2289:  User uploaded profile picture/avatar flickers during page load

### PR DESCRIPTION
**Pull request by labderahul1**

### CODE REVIEW

File: src/utils/normalize-mime-parts.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export function getProfileImageUrl(
 ) {
 	return `${origin}/service/home/~/?max_width=100&max_height=100&auth=${
 		jwtToken ? 'jwt' : 'co'
-	}&id=${encodeURIComponent(profileImageId)}&t=${Date.now()}${
+	}&id=${encodeURIComponent(profileImageId)}${
```
Comment (silentsakky) [inline src/utils/normalize-mime-parts.ts]: this means we have removed cache busting part, is this going to work even when images gets updated?

Comment (labderahul1) [reply src/utils/normalize-mime-parts.ts]: Yes. For new image generate new/unique profileImageId.

Comment (itsmeprashant) [reply src/utils/normalize-mime-parts.ts]: @silentsakky do you want to keep this change?

Comment (itsmeprashant) [reply src/utils/normalize-mime-parts.ts]: does this gernerate new id for updated images?

Comment (silentsakky) [reply src/utils/normalize-mime-parts.ts]: if we are having different profileImageId when updating image then it is fine to remove cache busting param

**Pull request merged**